palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/web/ResourceClient.java

```diff
@@ -27,22 +27,21 @@
 import uk.gov.gchq.palisade.service.palisade.request.GetResourcesByTypeRequest;
 
 import java.util.Map;
-import java.util.concurrent.CompletableFuture;
 
 @FeignClient(value = "resource-service", url = "${web.client.resource-service}")
 public interface ResourceClient {
 
```

Comment:
I question the merits of returning a synchronous function for all of the ResourceService - some of these resources could take a long time to complete.

Spring guide here on implementing async methods:
https://spring.io/guides/gs/async-method/

Any test harness that uses the async method will have to do an asyncDispatch() to ensure that the async call is completed.